components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -101,10 +102,24 @@ export default class CustomTimeRangeSelector extends React.Component {
 
     handleStartTimeChange(date) {
         this.startTime = date;
+        const start = Moment(this.startTime);
+        const end = Moment(this.endTime);
+        if(end.diff(start) < 0) {
```

Comment:
formatting issues

Reply:
This code is refactored with PR https://github.com/wso2/analytics-solutions/pull/270